config/config.yaml

```
@@ -29,17 +29,10 @@ ref:
-    # URL to pangenome haplotypes (vcf-file)
-    # Graph resources v1.1: https://github.com/human-pangenomics/hpp_pangenome_resources/
-    # Graph resources v1.0: https://github.com/human-pangenomics/hpp_pangenome_resources/blob/main/hprc-v1.0-mc.md
-    # Important: ensure that the haplotype vcf is built against the same reference genome as specified above under build
-    vcf: https://s3-us-west-2.amazonaws.com/human-pangenomics/pangenomes/freeze/freeze1/minigraph-cactus/hprc-v1.1-mc-grch38/hprc-v1.1-mc-grch38.raw.vcf.gz
-    # optional: In case contigs of the hoplotype vcf need to be renamed
-    # a list of python expressions can be defined.
-    # Contigs can be accessed by `contig`.
-    # Each expression will be successively saved into `renamed_contig`
-    rename_expressions:
-      - "'MT' if contig == 'chrM' else contig[3:]"
+    # Pangenome source and version: Only 'hprc' and v1.1 are supported for now. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)